core-services/prow/03_deployment/admin_config_updater_rbac.yaml

```diff
@@ -491,4 +491,32 @@ items:
   subjects:
   - kind: ServiceAccount
     name: config-updater
-    namespace: ci
\ No newline at end of file
+    namespace: ci
+- kind: Role
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: config-updater
+    namespace: release-controller-test-job
```

Comment:
The change above says the namespace is called `release-controller-test`.

Reply:
I'll be adding 3 namespaces in total:

1. release-controller-test
2. release-controller-test-job
3. release-controller-test-release

For the shared access, I just need the first 2 (test and job) to get the shared pieces. 

I assume this means I need to get the namespace additions, from my other PR, first.  Then, this PR, and lastly, my other PR.  Does that sound correct?

Reply:
Created a new PR with the namespace additions only:
https://github.com/openshift/release/pull/5317

Reply:
Namespace changes have been merged to master.